src/app/common.service.ts

```diff
@@ -12,15 +12,15 @@ export class CommonService {
   constructor(private http: Http) { }
 
   saveUser(user) {
-    return this.http.post('http://localhost:8080/api/SaveUser/', user)
+    return this.http.post('https://murmuring-refuge-48696.herokuapp.com/api/SaveUser/', user)
```

Comment:
Can you update to be the systems.io heroku url? And create an issue to not have this hard coded?

Reply:
Sure. Done :)